RELEASE/scripts/autoscend/paths/auto_path_util.ash

```diff
@@ -185,6 +185,11 @@ boolean auto_buySkills()  // This handles skill acquisition for general paths
 		{
 			visit_url("guild.php?action=buyskill&skillid=6", true);
 		}
+		if((my_level() >= 12) && (my_meat() >= 500) && !have_skill($skill[Deft Hands]) && 
+		(!have_skill($skill[Ambidextrous Funkslinging]) || (item_amount($item[beehive]) == 0 && item_amount($item[Time-Spinner]) == 0)))	//safe flyering
+		{
```

Comment:
I think this if statement is a bit overly complex.
maybe add a couple of booleans before it which it can use instead to resolve the more complex partitioned off OR / AND logic? like:

```suggestion
		boolean noStaggerItem = item_amount($item[beehive]) == 0 && item_amount($item[Time-Spinner]) == 0;
		boolean cantStagger = noStaggerItem || !have_skill($skill[Ambidextrous Funkslinging]);
		if((my_level() >= 12) && (my_meat() >= 500) && !have_skill($skill[Deft Hands]) && cantStagger)	//safe flyering
		{
```

also. what does deft hands have to do with this?